tests/base.py

```diff
@@ -0,0 +1,11 @@
+from unittest import TestCase
+
+
+########################################################################
+class BaseTinyFatTest(TestCase):
+    ####################################################################
+    def insert_entries(self, entries):
+        eids = self.db.insert_multiple(entries)
+        for entry in entries:
+            entry.update({"eid": eids.pop(0)})
+        return entries
```

Comment:
I figured that this was basic enough of a function to allow all potential tests to use